tests/system/test_fim/__init__.py

```diff
@@ -0,0 +1,24 @@
+# copyright: Copyright (C) 2015-2021, Wazuh Inc.
+#           Created by Wazuh, Inc. <info@wazuh.com>.
+#           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+from wazuh_testing.tools import WAZUH_LOGS_PATH
+
+
+def create_folder_file(host_manager, folder_path):
+    # Create folder
+    host_manager.run_command('wazuh-agent1', f'mkdir {folder_path}')
+
+    # Create file
+    host_manager.run_command('wazuh-agent1', f'touch {folder_path}/{folder_path}.txt')
+
+
+def wait_for_fim_scan_end(HostMonitor, inventory_path, messages_path, tmp_path):
+    HostMonitor(inventory_path=inventory_path,
+                messages_path=messages_path,
+                tmp_path=tmp_path).run()
+
+
+def clean_logs(host_manager):
+    host_manager.clear_file(host='wazuh-manager', file_path=os.path.join(WAZUH_LOGS_PATH, 'ossec.log'))
+    host_manager.clear_file(host='wazuh-agent1', file_path=os.path.join(WAZUH_LOGS_PATH, 'ossec.log'))
```

Comment:
Could you use `clean_environment` method, please?

Reply:
Done in commit [91ef4](https://github.com/wazuh/wazuh-qa/pull/2443/commits/91ef4414919fde73ccf875a29df9990852708b91)